<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NASettings</class>
 <widget class="QDialog" name="NASettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="naprog.qrc">
    <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>611</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="locale">
    <locale language="C" country="AnyCountry"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="C" country="AnyCountry"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Fonts</string>
    </attribute>
    <widget class="QFontComboBox" name="NAfont">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>268</width>
       <height>26</height>
      </rect>
     </property>
     <property name="fontFilters">
      <set>QFontComboBox::MonospacedFonts</set>
     </property>
     <property name="currentFont">
      <font>
       <family>Courier New</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Default Font</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Colors:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Keywords</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>62</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pitches </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Comments</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Translation</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>System</string>
    </attribute>
    <widget class="QLabel" name="heading">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Program paths</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="latexE">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>471</width>
       <height>22</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="C" country="AnyCountry"/>
     </property>
     <property name="placeholderText">
      <string>LaTeX binaries-Pfad</string>
     </property>
    </widget>
    <widget class="QLabel" name="latexL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LaTeX bin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gregorioE">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>471</width>
       <height>22</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="C" country="AnyCountry"/>
     </property>
     <property name="placeholderText">
      <string>Gregorio-Pfad</string>
     </property>
    </widget>
    <widget class="QLabel" name="gregorioL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gregorio</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lilypondE">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>471</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Lilypond-Pfad</string>
     </property>
    </widget>
    <widget class="QLabel" name="lilypondL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lilypond</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="naprog.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NASettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NASettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
